crates/api_models/src/events/locker_migration.rs

```diff
@@ -0,0 +1,9 @@
+use common_utils::events::ApiEventMetric;
+
+use crate::locker_migration::MigrateCardResponse;
+
+impl ApiEventMetric for MigrateCardResponse {
+    fn get_api_event_type(&self) -> Option<common_utils::events::ApiEventsType> {
+        Some(common_utils::events::ApiEventsType::RustLocker)
+    }
```

Comment:
I understand how this is needed, but can we have a generic event, for any auxiliary procedures that we are supposed to execute.
what do you think @lsampras ?